roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -55,6 +55,11 @@ nodelocaldns_ds_nodeselector: "kubernetes.io/os: linux"
 nodelocaldns_prometheus_port: 9253
 nodelocaldns_secondary_prometheus_port: 9255
 
+# nodelocaldns_additional_configs adds any extra configuration to coredns
+# nodelocaldns_additional_configs: |
+#   whoami
```

Comment:
See:https://github.com/kubernetes/kubernetes/blob/d9b95ea94f41b942440adbdfdbe35fb78a2ed6f7/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml#L58

By default, we don't need these configs indeed, Can you explain why we need it? thanks!

Reply:
For my particular use case, I wish to use the `template` plugin to respond to a query, so that this can be rewritten to a k8s service address via CNAME. I'm unable to utilise the existing `coredns_additional_configs` variable for this as this isn't used in the nodelocaldns config template, only in the coredns config template.

Reply:
I see, thanks for the explanation.